packages/oui-criteria-adder/src/index.spec.js

```diff
@@ -160,6 +160,119 @@ describe("ouiCriteriaAdder", () => {
             });
         });
 
```

Comment:
Nice tests!
It might be interesting to test if operators are well populated and if the value has the right component(calendar, number..)